tests/agent/nodejs/Makefile

```diff
@@ -6,4 +6,4 @@ stop:
 	docker-compose down -v
 
 setup:
-	npm install elastic-apm --save
+	npm install elastic/apm-agent-nodejs --save
```

Comment:
- Since there's no `package.json` in the `tests/agent/nodejs` folder, we should remove the `--save` option
- The node agent have now been deployed to npm as `elastic-apm`, so we should consider if we want this test to always take the latest stable version from npm (in which case it should be changed to `npm install elastic-apm`), or if we always want the latest HEAD from master in which case we can leave it as is